### PR TITLE
feat(docs): doc support table

### LINKED ISSUE #2809: [Feature] univer doc add insert tables
### 初始清单

- [X] 这真的是个问题吗？
- [X] 我已经在 Github Issues 中搜索过了，但没有找到类似的问题。

### 问题

univer doc 添加支持 插入表格，可调整行数和列数，以及表格内 单元格样式设置，合并/拆分单元格等功能

### 解决方案

新版本支持

### PR DESCRIPTION
**Pull request by Jocs**
close #2809

## Pull Request Checklist

- [x] Related tickets or issues have been linked in the PR description (or missing issue).
- [x] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [x] Unit tests have been added for the changes (if applicable).
- [x] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

File: packages/engine-render/src/components/docs/layout/doc-skeleton.ts [hunk 1]
```diff
@@ -67,6 +74,43 @@ export interface IFindNodeRestrictions {
     segmentPage: number;
 }
 
+function getPagePath(page: IDocumentSkeletonPage) {
+    const path: (string | number)[] = [];
+
+    // eslint-disable-next-line ts/no-explicit-any
```
Comment (DR-Univer) [inline packages/engine-render/src/components/docs/layout/doc-skeleton.ts:80]: any 还是少用

**Review by DR-Univer — APPROVED** [review]

File: packages/engine-render/src/components/docs/layout/doc-skeleton.ts (hunk 1 above)
Comment (Jocs) [reply packages/engine-render/src/components/docs/layout/doc-skeleton.ts:80]: 嗯嗯，后面改一下，any 还是限制在函数内的，对外面不会有影响

**Comment by wzhudev** [comment]
update-snapshots

**Pull request merged**